src/main/java/com/example/bookstore/repository/book/BookRepository.java

```diff
@@ -6,7 +6,9 @@
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
 
+@Repository
```

Comment:
actually, it's not necessary if you are extending Spring Data JPA repositories

Reply:
Not resolved